src/Controller/AdminController.php

```diff
@@ -52,6 +52,10 @@
  */
 class AdminController
 {
+    const ADMIN_DATETIME_FORMAT = 'Y-m-d H:i (e)';
+    const NOTIFICATION_STATUS_PROCESSED = 'PROCESSED';
+    const NOTIFICATION_STATUS_PENDING = 'PENDING';
```

Comment:
Could you move this to the `NotificationEntity` class, since I think it's more relevant to have it there.

Reply:
Agree :)